examples/modules/fetch-newrelic-api-access-key/providers.tf

```diff
@@ -0,0 +1,17 @@
+terraform {
+  required_providers {
+    newrelic = {
+      source = "newrelic/newrelic"
+    }
+    graphql = {
+      source = "sullivtr/graphql"
+       version = "2.5.4"
```

Comment:
@vinay-newrelic We can remove this version constraint, as this would not consumers of this module use the latest version of `sullivtr/graphql`